interface/models.py

```diff
@@ -89,13 +89,13 @@ class Submission(models.Model):
                                    on_delete=models.CASCADE,
                                    null=True)
     user = models.ForeignKey(User, on_delete=models.PROTECT, null=True)
-    stdout = models.CharField(max_length=32768,
+    stdout = models.TextField(max_length=32768,
                               default='',
                               blank=True)
-    stderr = models.CharField(max_length=32768,
+    stderr = models.TextField(max_length=32768,
```

Comment:
Q: Would the ```stderr`` field still be needed?

Reply:
No, but we should remove it in a separate PR.